site/assets/js/application.js

```diff
@@ -29,19 +29,6 @@
     checkbox.indeterminate = true
   })()
 
-  makeArray(document.querySelectorAll('.js-sidenav-group'))
```

Comment:
@ffoodd Can you check if `js-sidenav-group` is used anywhere else and if not remove it from the HTML too?

Reply:
Ah, NVM, you already have :)